utilities/serviceutils.py

```diff
@@ -103,8 +103,7 @@ def __hash__(self):
         """
         return hash(self.id) >> 22
 
-    async def send(self, content):
-        raise NotImplementedError()
+    async def send(self, *args, **kwargs) -> Any: ...
```

Comment:
why not `-> None` or whatever should be the actual type?

Reply:
`NoReturn` I think
https://docs.python.org/3/library/typing.html#typing.NoReturn

Reply:
we could create a alias which is union of the different types we expect for this?

Reply:
created alias